orangecontrib/geo/widgets/owmap.py

```diff
@@ -17,7 +17,10 @@
 from Orange.data import Table, Domain, TimeVariable, DiscreteVariable, ContinuousVariable
 from Orange.widgets import gui, widget, settings
 from Orange.widgets.utils.itemmodels import DomainModel
-from Orange.widgets.utils.webview import WebviewWidget, wait
+try:
+    from orangewidget.utils.webview import WebviewWidget, wait
+except:
```

Comment:
Usually you want to write `except ImportError`, see how it was handled here: https://github.com/biolab/orange3-text/commit/d2fab8ac111394574bfd2c668263bbe081bf956b

Reply:
Solved.